job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/triggers/TriggerContext/buildResult.groovy

```diff
@@ -0,0 +1,25 @@
+job('example') {
+    triggers {
+        buildResult('H/10 * * * *') {
+            combinedJobs()
+            triggerInfo('My job 1') {
```

Comment:
I when to say mandatory rules, I talk about BuilldResult Plugin:
- `triggerInfo` can be defined without defined `BuildResult` and no `BuildResult` is defined in this case.
- I defined a `BuildResult` default state only when `buildResult()` is calling.
`BuildResult... buildResults = [BuildResult.SUCCESS]`

But I do the refactoring.